### DISCUSSION TITLE
Collapse max height

### DISCUSSION THREAD

**Discussion by myregistration** [alpinejs/alpine#3007] [1. Help]
Is there a way to set the max height for the collapse?  I want to auto display a scrollbar for overflow content.  It's not working as expected, it's scrolling all the way up and the content is hidden.  Thanks!

**Comment by SimoTod** [comment]
Are you after the height when it's open or collapsed? If it's the first one, you can set max height via css, if it's the latter you have the min modifier. However, bear in mind that x-collapse also applies overflow hidden when the div is collapsed.

**Reply by myregistration** [reply]
Thanks for your response!  I did try setting a max height and h-full via tailwind css, but didn't work.  I got it to function the way I want using the following code on the element containing the  content.  I also had to set inset-0 overflow-hidden on it's parent container.  I was just hoping I could use collapse.  This code makes it open upward and I have a scrollbar for the content.


x-transition:enter="transform transition ease-in-out duration-500 sm:duration-700"
                 x-transition:enter-start="translate-y-full"
                 x-transition:enter-end="translate-y-0"
                 x-transition:leave="transform transition ease-in-out duration-500 sm:duration-700"
                 x-transition:leave-start="translate-y-0"
                 x-transition:leave-end="translate-y-full"

**Reply by SimoTod** [reply]
It looks like it does consider max-height however it reset the overflow value (see https://codepen.io/SimoTod/pen/dymyXvB) so yeah, you can't use collapse for it right now.

**Reply by myregistration** [reply]
Thanks for looking into it!  I hope they fix this eventually if it's a bug because it's much simpler to use collapse.  At least there is an alternative solution :)
